src/components/layout/nav.js

```diff
@@ -199,18 +197,19 @@ const Nav = () => {
                         </NavLink>
                     </NavCenter>
                     <NavRight>
-                        <NavRightContainer enable_move={show_button}>
-                            <NavButton onClick={handleLogin} primary>
-                                <span>{localize('Log in')}</span>
-                            </NavButton>
-                            <NavButton
-                                secondary
-                                movable_button
-                                onClick={toggleModal}
-                            >
-                                <span>{localize('Try for free')}</span>
-                            </NavButton>
-                        </NavRightContainer>
+                        <Button onClick={handleLogin} primary>
+                            <span>{localize('Log in')}</span>
+                        </Button>
+                        <SignupButton
+                            secondary
+                            onClick={toggleModal}
+                            move={show_button}
+                            ref={button_ref}
+                            width={button_ref}
```

Comment:
We only need to pass the ref once and then use the ref inside signupButton to get the width